test/UserRepository-test.js

```diff
@@ -0,0 +1,46 @@
+const chai = require('chai');
+const expect = chai.expect;
+const sampleData = require('../test/sampleData');
+const UserRepository = require('../src/UserRepository');
+
+describe('UserRepository', function() {
+  let userRepo;
+
+  beforeEach(function() {
+    userRepo = new UserRepository(sampleData);
+  });
+
+  it('should be a function', function() {
+    expect(UserRepository).to.be.a('function');
+  });
+
+  it('should instantiate a UserRepository', function() {
+    expect(userRepo).to.be.an.instanceof(UserRepository);
+  });
+
+  it('should hold all the user objects', function() {
+    expect(userRepo.allUsers).to.be.a('array');
+    expect(userRepo.allUsers[0].name).to.equal("Luisa Hane");
+  });
```

Comment:
Looking at your sample data file, it could be good to test here as well that the length of the array is 3 objects. 
Should hold all the user objects:
- Are we working with the right data type? (Expect to be array): ✅
- Do we have the specific objects that we want? (Testing a sample name): ✅
- Do we have ALL of the objects we expect to be in the array? (Expect length to be three)

Reply:
Not completely necessary but makes your test case a bit more robust and fully examines what you're prying at.